_docs_use-cases/use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer.md

```diff
@@ -0,0 +1,60 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer
+published: true
+title: "Delete a peer shared Attribute and notify peer"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate Enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA24
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: deletePeerSharedAttributeAndNotifyPeer
+  - description:
+  - feature category: Cross-identity attribute sharing
+  - tech category: Attributes
+  - status: DONE
+  - documentation status: DONE
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex:
+  - published: default
+  - link: use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer
+require:
+required_by:
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) a peer shared with you.
```

Comment:
I'd reformulate like that: "This use-case allows you to delete a peer shared Attribute, which is a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) that was created as a result of one of your peers sharing an Attribute with you."